Sources/Core/Transaction/Policies.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Policies.swift
+//  
+//
+//  Created by Jann Driessen on 01.11.22.
+//
+
+import Foundation
+import BigInt
+
+public typealias NoncePolicy = BlockNumber
+
+public enum GasLimitPolicy {
+    case automatic
+    case manual(BigUInt)
+    case limited(BigUInt)
+    case withMargin(Double)
```

Comment:
What does the value in the margin case mean exactly?
Some docs for that case would be very much appreciated 🙂